apps/frontend/src/shared/i18n/locales/zh/settings.json

```
@@ -0,0 +1,621 @@
+    "gitPath": "Git 路径",
+    "gitPathDescription": "Git 可执行文件路径(留空以自动检测)",
+    "gitPathPlaceholder": "git (默认)",
+    "githubCLIPath": "GitHub CLI 路径",
+    "githubCLIPathDescription": "GitHub CLI (gh) 可执行文件路径(留空以自动检测)",
+    "githubCLIPathPlaceholder": "gh (默认)",
+    "claudePath": "Claude CLI 路径",
+    "claudePathDescription": "Claude CLI 可执行文件路径(留空以自动检测)",
+    "claudePathPlaceholder": "claude (默认)",
+    "detectedPath": "自动检测",
+    "detectedVersion": "版本",
+    "detectedSource": "来源",
```

Biome (2.1.2)

[error] The key theme was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)